provisioning/transport/mqtt/src/mqtt.ts

```diff
@@ -35,7 +35,11 @@ export class Mqtt extends EventEmitter implements X509ProvisioningTransport, Sym
   private _subscribed: boolean;
 
   private _operations: {
-    [key: string]: (err?: Error, payload?: any) => void;
+    [key: string]: {
+      handler: (err?: Error, payload?: any, pollingInterval?: number) => void,
+      retryingString: string,
```

Comment:
>retryingString [](start = 6, length = 14)

this is used as the status of the current operation. the name "retryingString" does not match how it's used at all.

Reply:
I will change the name to statusString.

---
In reply to: [282204400](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282204400) [](ancestors = 282204400)